trillian_admin_api.proto

```diff
@@ -20,7 +20,7 @@ option java_outer_classname = "TrillianApiProto";
 
 package trillian;
 
-import "trillian.proto";
+import "github.com/google/trillian/trillian.proto";
```

Comment:
1. :)
1. I'm not sure if that will work due to Go package names. If it does, I actually prefer it as well.
1. Yes, I'd be OK with that.